examples/cis2-nft/src/lib.rs

```diff
@@ -418,11 +418,11 @@ fn contract_transfer<S: HasStateApi>(
         // If the receiver is a contract, we add sending it a message to the list of
         // actions.
         if let Receiver::Contract(address, function) = to {
-            let parameter = OnReceivingCis1Params {
+            let parameter = OnReceivingCis2Params {
                 token_id,
                 amount,
                 from,
-                contract_name: OwnedContractName::new_unchecked(String::from("init_CIS1-NFT")),
+                contract_name: OwnedContractName::new_unchecked(String::from("init_CIS2-NFT")),
```

Comment:
Is it still necessary to include the contract name? Wasn't that just for the callback?